client/src/Components/Organization/OrganizationDetail/ProjectDetails/ProjectDescription.jsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import Typography from "@material-ui/core/Typography";
+import Box from "@material-ui/core/Box";
+import List from '@material-ui/core/List';
+import { Link } from "@material-ui/core";
+
+const useStyles = makeStyles({
```

Comment:
Are you talking about moving Usestyles to a different file?

If Yes, then I wanted to ask - 
Since most of the specific element related styling is either one liner two, would be better if I use inline css styling instead of class selectors?

Reply:
That could have been fine if we wanted to provide styles at a few places and did not want to create a new file for it. But for the situation here I think inline styles would obfuscate our jsx code, also we have several classes that repeat multiple times like `head`  `text`  `github` etc and repeating the inline styles for them is way too much repetition in my opinion. If we want to go with inline styling then I think the best approach would be create HOCs where it makes sense for components which are getting repeated multiple times.

Reply:
@ksraj123 what I meant is - wherever there is one two css properties to add then I think Inline is fine. 
But whenever a certain set of css properties are repeating, I have created class for it -> now a separate file as well.

Reply:
Yeah, that would be fine. For one or two line styles which are not getting repeated, you can make it inline, move everything else to a different file.